paperless-ngx/rootfs/etc/cont-init.d/90-cron-backup.sh

```diff
@@ -0,0 +1,30 @@
+#!/command/with-contenv bashio
+# shellcheck shell=bash
+set -e
+
+echo "[INFO] Configuring backup cron job..."
+
+if bashio::config.has_value 'backup_enabled'; then
```

Comment:
Those conditions seems useless to me since you can have default values in supervisor 
I do not use those checks I assume it's always defined